README.md

```diff
@@ -187,10 +188,60 @@ with dag:
     my_second_sql_transformation(my_table)
 ```
 
+### The output_table parameter
```

Comment:
Are we planning on adding more here, or was this intended to be a heading one level higher?

Reply:
Ah good catch thank you